pages/styles/events.css

```diff
@@ -36,8 +16,14 @@
   flex-direction: column;
 }
 
-.meetupCard div > div:first-child {
-  display: none;
+.meetupCard > div > div:nth-child(2) {
```

Comment:
You already defined `text-align: center` within the context of the FlatCard... So, if you need it here as well, there's something wrong on that end. Maybe apply it using `.flatCardButton > button` (etc) like I used below, instead.

The `margin-bottom: -3.6em` is exactly what I was hoping to get away from with putting the button into the context of the FlatCard.

I'm okay leaving this here for now, because it'll be a more complex styling, but I'd like to see if we can use CSS within the FlatCard to _always_ have the bottom Button/LinkButton show up where it is here (regardless of content).

Reply:
The `margin-bottom: -3.6em` (`-2em` for the padding, plus `-1.6em` to center) was to position the button over the bottom border. I wouldn't consider it "bad", and I'm not sure how else you would do this that wouldn't involve negative spacing. Dealing with `position: absolute` would seem like a bigger PITA and still require negative spacing.

We could reduce the amount of negative spacing necessary by eliminating the `2em` padding added by the `borderContainer` class.

For text-alignment on FlatCard button: you're right I missed that, and about to push update.

Reply:
You could do position relative off of the content of the card and have it be a constant number. I didn’t call it bad, but I would hope we don’t have to do it because it’s a bigger PITA to have to change styling based upon content.

Reply:
I should've expressed that better, didn't imply you were calling it bad but that doing it that way with negative spacing wouldn't be "incorrect".

I also should've just tested that out before opening my yapper. Turns out I had an unnecessary flex styling that I removed and using position absolute didn't require negative spacing for the button. I just pushed a change for moving those updates to FlatCard too while I was at it.